packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -2,7 +2,7 @@
 
 import type { Compiler, container } from 'webpack';
 import type { ModuleFederationPluginOptions } from '../types';
-import { extractUrlAndGlobal } from '@module-federation/utilities';
+import { extractUrlAndGlobal } from '@module-federation/utilities/src/utils/pure';
```

Comment:
move pure utils to pure folder for smaller footprint of code moved into runtime module as eager